unattended_installer/install_functions/installCommon.sh

```diff
@@ -617,4 +617,32 @@ function installCommon_startService() {
         exit 1
     fi
 
+}
+
+function installCommon_yumInstallList(){
+
+    dependencies=("$@")
+    not_installed=()
+    for dep in "${dependencies[@]}"; do
+        if [ "${dep}" == "openssl" ]; then
+            if ! yum list installed 2>/dev/null | grep -q -E ^"${dep}\.";then
```

Comment:
Is this exception still necessary?

Reply:
This has been reworked. Please see https://github.com/wazuh/wazuh-packages/issues/2047#issuecomment-1434631561 and https://github.com/wazuh/wazuh-packages/issues/2047#issuecomment-1436826125